lib/compilers/argument-parsers.ts

```diff
@@ -844,8 +844,8 @@ export class RustParser extends BaseParser {
     }
 
     static override async getPossibleEditions(compiler: BaseCompiler): Promise<string[]> {
-        const result = await compiler.execCompilerCached(compiler.compiler.exe, ['--help']);
-        const re = /--edition ([\d|]*)/;
+        const result = await compiler.execCompilerCached(compiler.compiler.exe, ['--help', '-v']);
+        const re = /--edition <?([\w|]*)>?/;
```

Comment:
Is the `future` key new from 1.88 then?

Reply:
`future` is perma-unstable (`nightly` only) and intended to receive features for a future edition when they’re ready, even if the next edition isn’t named yet (similar to `-std=c++2x` for GCC/clang), see https://github.com/rust-lang/rust/blob/243c5a35e18b2634892fe7091d5ee888a18f77f5/compiler/rustc_span/src/edition.rs#L26-L35 and https://github.com/rust-lang/rust/pull/137606.

I don’t know if there’s currently any difference to 2024, so including `future` is currently mostly for completeness’ sake.